.github/workflows/build_release.yml

```
@@ -0,0 +1,175 @@
+  build:
+    name: "Build Release"
+    runs-on: macos-15
+    env:
+      XCODE_PATH: "/Applications/Xcode_16.app"
+      # These variables are defined in the repository settings
+      PROJECT_NAME: ${{ vars.PROJECT_NAME }}
+      INSTALLER_CERTIFICATE: ${{ vars.DEV_INSTALLER_CERT_CN }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4 # v4
```

yamllint

[warning] too few spaces before comment

(comments)